bin/node/runtime/Cargo.toml

```diff
@@ -14,59 +14,59 @@ rustc-hex = { version = "2.0", optional = true }
 serde = { version = "1.0.102", optional = true }
 
 # primitives
-sp-authority-discovery = { path = "../../../primitives/authority-discovery", default-features = false }
-sp-consensus-babe = { path = "../../../primitives/consensus/babe", default-features = false }
+sp-authority-discovery = { version = "2.0.0", default-features = false, path = "../../../primitives/authority-discovery" }
+sp-consensus-babe = { version = "0.8", default-features = false, path = "../../../primitives/consensus/babe" }
 sp-block-builder = { path = "../../../primitives/block-builder", default-features = false}
```

Comment:
Why is it missing here?

Reply:
I suppose this didn't match because of a missing space at the end...

Reply:
not intentional.